tests/components/modbus/test_init.py

```diff
@@ -863,6 +863,7 @@ async def test_shutdown(
     assert caplog.text == ""
 
 
+@pytest.mark.skip(reason="disabled because the test is flapping")
```

Comment:
It would be better to solve the problem:
either:
- remove the assert
or
- test for UNKNOWN and UNAVAILABLE 


the test is important for the stability of the modbus integration.

Apart from that, I believe this will cause a code coverage problem, but lets see.